samples/boards/nrf/nrfx_prs/src/main.c

```diff
@@ -288,9 +288,7 @@ static bool background_transfer(const struct device *spi_dev)
 	static const uint8_t tx_buffer[] = "Nordic Semiconductor";
 	static uint8_t rx_buffer[sizeof(tx_buffer)];
 	static const struct spi_cs_control spi_dev_cs_ctrl = {
-		.gpio_dev = DEVICE_DT_GET(DT_GPIO_CTLR(SPI_DEV_NODE, cs_gpios)),
-		.gpio_pin = DT_GPIO_PIN(SPI_DEV_NODE, cs_gpios),
-		.gpio_dt_flags = DT_GPIO_FLAGS(SPI_DEV_NODE, cs_gpios)
+		.gpio = GPIO_DT_SPEC_GET_BY_IDX(SPI_DEV_NODE, cs_gpios, 0),
```

Comment:
```suggestion
		.gpio = GPIO_DT_SPEC_GET(SPI_DEV_NODE, cs_gpios),
```

Reply:
I guess we can go even further and use 

<pre>
.gpio = SPI_CS_GPIOS_DT_SPEC_GET(SPI_DEV_NODE),
</pre>

but the current approach is also fine.